packages/components/src/__future__/Tag/_docs/RemoveableTag.mdx

```diff
@@ -0,0 +1,67 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as RemoveableTagStories from "./RemoveableTag.stories"
+
+<Meta of={RemoveableTagStories} />
+
+# RemoveableTag
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/__future__/RemoveableTag"
+  designGuidelines="https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%F0%9F%92%9C-UI-Kit%3A-Heart?type=design&node-id=48986-137441&mode=design&t=9o7AOjKCbdVADUkP-0"
```

Comment:
Remove this since it's the wrong docs and we don't want to expose the future ones.